target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr802n-v2.c

```diff
@@ -0,0 +1,98 @@
+/*
+ *  TP-LINK TL-WR802N v2
+ *
+ *  Copyright (C) 2016 Thomas Roberts <tom.p.roberts@gmail.com <mailto:tom.p.roberts@gmail.com>>
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License version 2 as published
+ *  by the Free Software Foundation.
+ */
+
+#include <linux/gpio.h>
+#include <linux/platform_device.h>
+
+#include <asm/mach-ath79/ath79.h>
+#include <asm/mach-ath79/ar71xx_regs.h>
+
+#include "common.h"
+#include "dev-eth.h"
+#include "dev-gpio-buttons.h"
+#include "dev-leds-gpio.h"
+#include "dev-m25p80.h"
+#include "dev-wmac.h"
+#include "machtypes.h"
+
+#define TL_WR802N_V2_GPIO_LED_SYSTEM	13
+#define TL_WR802N_V2_GPIO_BTN_RESET	11
+
+#define TL_WR802N_V2_KEYS_POLL_INTERVAL	20	/* msecs */
+#define TL_WR802N_V2_KEYS_DEBOUNCE_INTERVAL	(3 * TL_WR802N_V2_KEYS_POLL_INTERVAL)
+
+static const char *tl_wr802n_v2_part_probes[] = {
+	"tp-link",
+	NULL,
+};
+
+static struct flash_platform_data tl_wr802n_v2_flash_data = {
+	.part_probes	= tl_wr802n_v2_part_probes,
+};
+
+static struct gpio_led tl_wr802n_v2_leds_gpio[] __initdata = {
+	{
+		.name		= "tp-link:green:system",
```

Comment:
Change led name accordance with the standard. Like board:color:funcion 
tl-wr802n-v2:green:system